lend/src/components/ActionMMModal/Footer.tsx

```diff
@@ -0,0 +1,328 @@
+import React from "react";
+import { Col, Row } from "react-bootstrap";
+import colors from "shared/lib/designSystem/colors";
+import styled, { keyframes } from "styled-components";
+import { components } from "../../designSystem/components";
+import { PrimaryText, Title } from "../../designSystem";
+import sizes from "../../designSystem/sizes";
+import { getEtherscanURI, VaultOptions } from "../../constants/constants";
+import { formatBigNumber } from "../../utils/math";
+import { getUtilizationDecimals } from "../../utils/asset";
+import { usePoolsAPR } from "../../hooks/usePoolsAPR";
+import { fadeIn } from "shared/lib/designSystem/keyframes";
+import { css } from "styled-components";
+import { BaseUnderlineLink } from "shared/lib/designSystem";
+import { useVaultsData } from "../../hooks/web3DataContext";
+import { useWeb3React } from "@web3-react/core";
+import ExternalLinkIcon from "../Common/ExternalLinkIcon";
+import UtilizationBar from "../Common/UtilizationBar";
+import currency from "currency.js";
+import { ActionModalEnum } from ".";
+
```

Comment:
Nit: You can actually reverse an animation via `animation-direction`, but not necessary